app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -188,13 +188,59 @@
                                     :id                => record.id,
                                     :prefix            => "amqp",
                                     :basic_info_needed => true}
+
+          .col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'vmware_cloud'"}
+            %div{"ng-if" => "emsCommonModel.event_stream_selection != 'amqp'"}
+              %div{"ng-init" => "emsCommonModel.event_stream_selection = 'none'"}
+            %label.radio-inline.control-label{"for" => "none_radio"}
+              %input{:type         => "radio",
+                     :name         => "event_stream_selection",
+                     :id           => "none_radio",
+                     "ng-checked"  => "emsCommonModel.event_stream_selection == none",
+                     :value        => "none",
+                     "ng-model"    => "emsCommonModel.event_stream_selection",
+                     "checkchange" => ""} None
+            %label.radio-inline.control-label{"for" => "amqp_radio"}
+              %input{:type         => "radio",
+                     :name         => "event_stream_selection",
+                     :id           => "amqp_radio",
+                     "ng-checked"  => "emsCommonModel.event_stream_selection == amqp",
+                     :value        => "amqp",
+                     "ng-model"    => "emsCommonModel.event_stream_selection",
+                     "checkchange" => ""} AMQP
+            %hr
+            %div{"ng-if" => "emsCommonModel.event_stream_selection == 'none'"}
+              Disable event monitoring.
+            %div{"ng-if" => "emsCommonModel.event_stream_selection == 'amqp'"}
+              = render :partial => "layouts/angular-bootstrap/endpoints_angular",
+                       :locals  => {:ng_show                => true,
+                                    :ng_model               => "#{ng_model}",
+                                    :id                     => record.id,
+                                    :prefix                 => "amqp",
+                                    :ng_reqd_api_port       => "false",
+                                    :ng_reqd_hostname       => "false"}
```

Comment:
@miha-plesko Besides the comment from @AparnaKarve, I am interested in why you had to set `ng_reqd_hostname` to `false`? For port I understand it's not required as a default can be used in most cases, but hostname should not be a default, I guess...